rickshaw.js

```diff
@@ -3215,6 +3222,18 @@ Rickshaw.Graph.RangeSlider.Preview = Rickshaw.Class.create({
     // Use the first graph as the "master" for the frame state
     var masterGraph = this.graphs[0]
 
+    var funcArray = masterGraph.updateCallbacks.map(function(func) {
+      return func.name === 'onUpdateCallback'
```

Comment:
This does not seem like an ideal way to check if the function is present. Is there a better way? Moving `onUpdateCallback` into `render`'s scope and checking `includes()` with the function directly produced a lot of false negatives.